src/common/options.cc

```diff
@@ -4430,6 +4430,19 @@ std::vector<Option> get_global_options() {
 
     Option("bluestore_rocksdb_cf", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
     .set_default(true)
+#ifdef WITH_SEASTAR
+    // This is necessary as the Seastar's allocator imposes restrictions
+    // on the number of threads that entered malloc/free/*. Unfortunately,
+    // RocksDB sharding in BlueStore dramatically lifted the number of
+    // threads spawn during RocksDB's init.
+    .set_validator([](std::string *value, std::string *error_message){
+      if (*value == "true" || *value != "0") {
+        *error_message = "invalid BlueStore configuration. disabled";
```

Comment:
s/disabled//

i actually posted a revised version, the original version was to change the value back to "false". like
```c++
if (*value == "true" || *value != "0") {
  *value = "false";
  *error_message = "invalid BlueStore configuration. disabled";
}
return 0;
```

but when posting my suggestion, i changed my mind. and thought that instead of changing user's preferences. it'd be better to keep at it is.

so unless you think we should change `value` on behalf of user, we cannot put "disabled" in the `error_message`.

Reply:
Just on the `return -<err>`: it looks Ceph's config subsystem doesn't support exiting on that. It's worth verification / implementing, I think. WIP (separate PR).